app/views/favorites/index.html.erb

```diff
@@ -0,0 +1,21 @@
+
+<div class="content">
+
+  <h1 class="header">Favorite Pets</h1>
+
+  <% @favorite_pets.each do |pet| %>
+  <div class="pet-list">
```

Comment:
because @favorite is called in the application controller you can access it on all pages without having it in the `favorite_controller def index...`